test/OffsetHelper.test.ts

```diff
@@ -121,7 +125,102 @@ describe("OffsetHelper", function () {
     });
   });
 
-  describe("Testing autoOffset()", function () {
+  const TOKEN_POOLS = [
+    { name: "NCT", token: () => nct },
+    { name: "BCT", token: () => bct },
+  ];
+
+  describe("#autoOffsetUsingToken(address,uint256,address)[fixed-in]", function () {
+    async function retireFixedInToken(
+      fromToken: IERC20,
+      fromAmount: BigNumber,
+      toToken: IToucanPoolToken
```

Comment:
I think a better name for this var would be `poolToken`

Reply:
definitely :) changed in both tests.